compiler/surface/lexer.cppo.ml

```diff
@@ -778,9 +778,6 @@ let lex_raw (lexbuf : lexbuf) : token =
     | _ -> (
         (* Nested match for lower priority; `_` matches length 0 so we effectively retry the
            sub-match at the same point *)
-        let lexbuf = lexbuf in
```

Comment:
Great!

Reply:
Moved to #626 because there is some silly CI breakage